src/core/MOM_forcing_type.F90

```diff
@@ -212,6 +223,19 @@ module MOM_forcing_type
     ice_fraction  => NULL(), &  !< fraction of sea ice coverage at h-cells, from 0 to 1 [nondim].
     u10_sqr       => NULL()     !< wind magnitude at 10 m squared [L2 T-2 ~> m2 s-2]
 
+  ! Forcing fields required for MARBL
+  real, pointer, dimension(:,:) :: &
+    noy_dep => NULL(),               & !< NOy Deposition [conc Z T-1 ~> conc m s-1]
+    nhx_dep => NULL(),               & !< NHx Deposition [conc Z T-1 ~> conc m s-1]
+    atm_co2 => NULL(),               & !< Atmospheric CO2 Concentration [ppm]
+    atm_alt_co2 => NULL(),           & !< Alternate atmospheric CO2 Concentration [ppm]
+    dust_flux => NULL(),             & !< Flux of dust into the ocean [R Z T-1 ~> kgN m-2 s-1]
+    iron_flux => NULL()                !< Flux of dust into the ocean [conc Z T-1 ~> conc m s-1]
+
+  real, pointer, dimension(:,:,:) :: &
+    fracr_cat   => NULL(),           & !< per-category ice fraction
```

Comment:
Please add a description of the units of `fracr_cat` and `qsw_cat` to the comments describing them.

Reply:
Addressed in 5402ce0